src/app/cli/src/init/test_block_command_verification.ml

```diff
@@ -0,0 +1,123 @@
+open Core_kernel
+open Async_kernel
+
+let time_block_command_verification ~(large_precomputed_json_file : string)
+    ~(log_directory : string) =
+  let signature_kind = Mina_signature_kind.t_DEPRECATED in
+  (* TODO: Should be replaced by some to_string method for the signature_kind,
+     an inverse of an of_string method that should also be created for the
+     signature kind refactor, when the user-facing side of that is figured
+     out. *)
+  let signature_kind_string =
+    match signature_kind with
+    | Mina_signature_kind.Mainnet ->
+        "mainnet"
+    | Mina_signature_kind.Testnet ->
+        "testnet"
+    | Mina_signature_kind.Other_network string ->
+        "other network: " ^ string
+  in
+  printf "Validating block with signature_kind \"%s\"\n" signature_kind_string ;
+  printf
+    "If validation fails you may need to compile the test with a different \
+     profile\n" ;
+  printf "Reading block from %s\n" large_precomputed_json_file ;
+  let json =
+    Yojson.Safe.from_string (In_channel.read_all large_precomputed_json_file)
+  in
+  let precomputed =
+    match Mina_block.Precomputed.of_yojson json with
+    | Ok json ->
+        json
+    | Error err ->
+        failwith err
+  in
+  let logger = Logger.create () in
+  let commit_id = "<not-needed-for-test>" in
+  Logger.Consumer_registry.register ~id:Logger.Logger_id.mina
+    ~processor:Internal_tracing.For_logger.processor ~commit_id
+    ~transport:
+      (Logger_file_system.dumb_logrotate ~directory:log_directory
+         ~log_filename:"internal-tracing.log"
+         ~max_size:(1024 * 1024 * 10)
+         ~num_rotate:50 )
+    () ;
+  let%bind () =
+    Internal_tracing.toggle ~commit_id ~force:true ~logger `Enabled
+  in
+  let proof_cache_db = Proof_cache_tag.For_tests.create_db () in
```

Comment:
Didn't know we alias `create_identity_db` as `For_tests.create_db` at all...
(This is a comment not a request change)